pkg/server/v3/gateway_test.go

```
@@ -100,7 +100,7 @@ func TestGateway(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-		resp, code, err := gtw.ServeHTTP(req)
+		resp, code, _ := gtw.ServeHTTP(req)
```

Let's not ignore the errors here, I'd rather add an error check